_compdemos/maxquant.md

```diff
@@ -0,0 +1,234 @@
+[Visual Studio Code](https://code.visualstudio.com/) is a good choice if you don't have a favorite.
+
+You need to make the following changes (note that in future, there may be an automated submission script that takes care of this for you, but for now it's a manual process).
+
+#### Convert Windows paths to Linux paths
+
+Search for `<filePaths>` in the `mqpar.xml` file to find the paths to your data files. Also search for `<fastaFilePath>` to find the path to your `.fasta` file.
+
+Note that the `X:` drive on Windows is equivalent to the `/fh/fast` filesystem on Linux.
+
+You need to convert all Windows paths to Linux, which means:
+
+* Replace the Windows drive letter (`C:`, `X:`, etc.) with `/fh/fast`.
+* Replace all backslashes (`\`) with forward slashes (`/`).
+
+**Example**: You may have paths like this in your `mqpar.xml` file:
+
+```xml
+   <filePaths>
+      <string>c:\mq-job\JK050120_050620_Ub_BioRep1.raw</string>
+      <string>c:\mq-job\JK050120_050620_Ub_BioRep2.raw</string>
+      <string>c:\mq-job\JK050120_050620_Ub_BioRep3.raw</string>
+   </filePaths>
+   ...
+    <fastaFilePath>c:\mq-job\uniprot-proteome_UP000005640.fasta</fastaFilePath>
+```
+
+If your PI is Jane Doe, you will want to change these paths to something like the following (replace `doe_j` with your actual PI's last name and first initial):
+
+```xml
+   <filePaths>
+      <string>/fh/fast/doe_j/proteomics/maxquant-jobs/job20200610/JK050120_050620_Ub_BioRep1.raw</string>
+      <string>/fh/fast/doe_j/proteomics/maxquant-jobs/job20200610/JK050120_050620_Ub_BioRep2.raw</string>
+      <string>/fh/fast/doe_j/proteomics/maxquant-jobs/job20200610/JK050120_050620_Ub_BioRep3.raw</string>
+   </filePaths>
+   ...
+ <fastaFilePath>/fh/fast/doe_j/proteomics/maxquant-jobs/job20200610/uniprot-proteome_UP000005640.fasta</fastaFilePath>   
+```
+
+Again, you need to make sure that the directory `/fh/fast/doe_j/proteomics/maxquant-jobs/job20200610/`, equivalent to `X:\doe_j\proteomics\maxquant-jobs\job20200610\` on Windows) actually exists and contains your `mqpar.xml` file, your data files, your `.fasta` file, and any other optional `.xml` files.
+
+#### Change the number of threads to match the number of data files
+
+The Windows-based pipeline took care of this step automatically, but it's a quick manual edit. Search for `<numThreads>` and change the its value to the number of 
+data files you have. 
+
+For example you may see this:
+
+```xml
+   <numThreads>1</numThreads>
+```
+
+If you have 24 data files in your job, change the number accordingly:
+
+```xml
+   <numThreads>24</numThreads>
+```
+
+Remember the number you changed it to; you will need it later.
+
+### Submit job to cluster
+
+The rest of these steps need to be done on a Linux HPC system. Using `ssh`, `PuTTY`, or `NoMachine`, connect to the system `rhino01`.  For more information about these connection methods, read [Computing Resource Access Methods](/scicomputing/access_methods/).
```

Comment:
The "rhino" alias can be used for ssh and putty, nomachine needs the explicit name.  Maybe punt and just remove ssh and putty statement leaving the link to the access methods page?

Reply:
Thanks. I still need to put in a machine name to connect to, somewhere. And I don't want to use "rhino" until that points to bionic machines. It's possible that people will use these instructions before the changeover on Tuesday.